samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/QuestionScoreListener.java

```diff
@@ -103,6 +104,8 @@
 	private static final String noAnswer = ContextUtil.getLocalizedString(MSG_BUNDLE, "no_answer");
 	private static final String noneOfTheAbove = ContextUtil.getLocalizedString(MSG_BUNDLE, "none_above");
 
+	private RubricsService rubricsService = ComponentManager.get(RubricsService.class);
```

Comment:
Use JSF's managed bean DI vs the static cover here.

Reply:
Sorry, I'm not sure about the precise dependency injection way you want here. Samigo beans are always doing it like this (not the most desirable, I concur) and I thought consistency within the tool could have priority against introducing different paths. Anyway, let me know your proposal and I'll fix it. And maybe we can create a jira to substitute all the other similar cases. Thanks!